Class/Practise 7.1/include/RectangleComparer.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include "Rectangle.hpp"
+#include <vector>
+
+class RectangleComparer {
+public:
+    static Rectangle getRectangleWithMaxArea(const std::vector<Rectangle>& rectangles);
```

Comment:
Minor: we can use some alias name for std::vector<Rectangle> to make it more readable
using RectangleList = std::vector<Rectangle>;